crates/biome_cli/src/execute/process_file/lint.rs

```diff
@@ -11,14 +11,26 @@ use std::path::Path;
 use std::sync::atomic::Ordering;
 
 /// Lints a single file and returns a [FileResult]
-pub(crate) fn lint<'ctx>(ctx: &'ctx SharedTraversalOptions<'ctx, '_>, path: &Path) -> FileResult {
+pub(crate) fn lint<'ctx>(
+    ctx: &'ctx SharedTraversalOptions<'ctx, '_>,
+    path: &Path,
+    suppress: bool,
+    suppression_reason: &Option<String>,
```

Comment:
It's idiomatic to pass an `Option<&>` instead of `&Option<>`. Use `.as_ref()` to retrieve a `Option<&>`

Reply:
I actually hate that I wrote this.